docs/source/pillars/information_governance.md

```diff
@@ -86,6 +86,12 @@ _The ability of the TRE organisation to monitor compliance with internal and ext
 * - You report on and share outcomes of each audit of your TRE organisation with the required bodies
   - This may included regulatory bodies or the organisations that manage accreditations you have
   - Mandatory
+* - You log user actions, network connection attempts, infrastructure actions and data access within a TRE
```

Comment:
These all seems a bit vague. @machintim do you know of an external reference or some existing standard text we could use instead?